### PR TITLE
[security certs] Build landing page

### LINKED ISSUE #9747: [security certifications] Build /security/certifications page
- [x] Landing page (/security/certifications) [copy doc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)

Please use feature branch `security-certifications` - (Make PR against this instead of master)

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Build landing page at `/security/certifications`
- Add secondary nav (aware these pages don't exist yet but this is a feature branch)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/security/certifications
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against [copy doc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit#) 
- This PR is made against the feature branch `security-certifications` - many of the links are pointing to pages that don't exist yet but will be built shortly 

## Issue / Card

Fixes #9747

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @bethcollins92, could we use the list with dividers for the second section please, so we get the horizontal lines.

I'm not sure the green and black ticks are different enough, could we have green and orange, or are we avoiding orange for a reason?

**Comment by bethcollins92** [comment]
Thanks @spencerbygraves, made suggested changes

**Comment by spencerbygraves** [comment]
@bethcollins92 great, thanks.

File: templates/security/certifications.html [hunk 1]
```diff
@@ -3,202 +3,203 @@
...
+   </div>
+   <div class="col-4 u-hide--small u-align--center">
+     {{
+       image(
+         url="https://assets.ubuntu.com/v1/7953a068-security-1.svg",
+         alt="",
+         height="300",
+         width="250",
+         hi_def=True,
+         loading="lazy",
```
Comment (petesfrench) [inline templates/security/certifications.html:26]:
```suggestion
         loading="auto",
```
Set images in the header to auto, everything below to lazy

File: templates/security/certifications.html [hunk 2]
```diff
@@ -3,202 +3,203 @@
...
+   </div>
+   <div class="col-4 p-divider__block">
+     <h3>Transparency</h3>
+     <p>Canonical is transparent on the Security Features we enable on Ubuntu. We have a <a href="/security/notices">public vulnerability disclosure policy</a> and known vulnerabilities are disclosed in <a href="/security/oval">Machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party tools.</p>
+     <a class="p-button--neutral p-link--external" href="https://wiki.ubuntu.com/Security/Features">See our security features</a>
+   </div>
+   <div class="col-4 p-divider__block">
+     <h3>Security certification and 3rd party attestation</h3>
+     <p>Access to certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and Ubuntu Pro provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
```
Comment (petesfrench) [inline templates/security/certifications.html:48]: Should 'Ubuntu Pro' also be a link here for consistency?

**Review by petesfrench — CHANGES_REQUESTED** [review]
Hey, just a couple of small things that I found but all good otherwise. Also it would let me add a comment in the code, but on line 193 there is a link missing on 'CIS hardening'.

File: templates/security/certifications.html (hunk 2 above)
Comment (bethcollins92) [reply templates/security/certifications.html:48]: Goos spot, makes sense!

**Comment by bethcollins92** [comment]
Thanks Pete - ready for re-review

**Review by petesfrench — APPROVED** [review]
LGTM :+1:

**Pull request merged**